### DISCUSSION TITLE
Serveur Infomaniak inaccessible

### DISCUSSION THREAD

**Discussion by Andrea-Graf** [web-classroom/web-classroom.github.io#61] [Q&A : Pratique / Labos] [answered]
Bonjour,

Le serveur Infomaniak n'est plus accessible, ce qui est un peu embêtant pour finir de trouver les flags 😟. Serait-il possible d'y jeter un coup d'œil ?

Merci d'avance et bon week-end.

**Comment by Lyrafll** [comment]
@web-classroom/web-vgx-teachers !
 Site down pour moi aussi 😄

**Comment by Nortalle** [comment]
Bonjour tout le monde, désolé, je vais essayer de voir ce queje peux faire, je n'ai que mon smartphone avec moi

**Comment by hadrylouis** [comment, marked as answer]
Bonjour à tous @web-classroom/web-vgx-students-tuesday @web-classroom/web-vgx-students,

Le serveur est à nouveau opérationnel. 

Désolé pour le délai d'attente et n'hésitez pas à nous recontacter si le problème survient à nouveau. Je me chargerai de faire des checks de temps en temps.

Bonne soirée à tous.
